orchagent/bufferorch.cpp

```diff
@@ -210,6 +210,12 @@ bool BufferOrch::isPortReady(const std::string& port_name) const
     return result;
 }
 
+void BufferOrch::clearBufferPoolWatermarkCounterCounterIdList(const sai_object_id_t object_id)
```

Comment:
It is required for reclaiming buffer. 
The zero buffer pool will be removed once the last admin down port is started up.
Originally, a buffer pool can not be removed cleanly because the counter id list was not removed. This is to support it.